app/assets/stylesheets/pages/_informations.scss

```diff
@@ -0,0 +1,48 @@
+.infos-wrapper {
+  max-width: 800px;
+  margin: auto;
+  max-height: 300px;
+  margin-bottom: 48px;
+}
+
+.infos-rewards h1 {
+  padding: 24px 0 16px 0;
+}
+
+.year-awards {
+  display: flex;
+}
+
+.year {
+  padding-right: 48px;
+  font-weight: bolder;
+}
+
+@media (min-width: 768px) {
```

Comment:
Not needed, it's the default behaviour.

Reply:
I keep it just for the columns : 2, so I don't have to overwrite it on mobile version.